tools/lib/find_add_class.py

```diff
@@ -93,6 +93,8 @@ def find(fns):
                         html_classes = ['error', 'success']
                     elif fn == 'ui_report.js' and 'status_classes' in line:
                         html_classes = ['alert']
+                    elif fn == 'modal.js':
+                        continue
```

Comment:
Ok - added list `EXCLUDED_MODULES` for specifying excluded JS files, updated to `find` to check if `fn in EXCLUDED_MODULES`.